src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -34,6 +35,8 @@ internal class GrpcWorker : IWorker
         private readonly IMethodInfoLocator _methodInfoLocator;
         private readonly IOptions<GrpcWorkerStartupOptions> _startupOptions;
         private readonly ObjectSerializer _serializer;
+        private readonly string _appLocation;
+        private Task<FunctionMetadataResponse> _functionMetadataResponse;
```

Comment:
naming nit: IMHO, Adding the "Task" suffix improves readability. This variable holds the task. ex : `_functionMetadataResponseLoadingTask`